code/src/components/BackgroundImage.css

```diff
@@ -0,0 +1,49 @@
+.background-image {
+  width: 100%;
+  height: 140vh;
+  opacity: 0;
+  object-fit: cover;
+  position: relative;
+  z-index: -1;
+}
+
+@keyframes fader {
+  0% {background: url(/images/tim-toomey-eiY4KJ62P5Q-unsplash.jpg); background-size: cover;}
+  12.5% {background: url(/images/pexels-wendy-wei-1190297.jpg); background-size: cover;}
+  25% {background: url(/images/aditya-chinchure-ZhQCZjr9fHo-unsplash.jpg); background-size: cover;}
+  37.5% {background: url(/images/anthony-delanoix-hzgs56Ze49s-unsplash.jpg); background-size: cover;}
+  50% {background: url(/images/joseph-pearson-FrmpLKLNgNw-unsplash.jpg); background-size: cover;}
+  62.5% {background: url(/images/maxime-lebrun-l8h1hXBvzkQ-unsplash.jpg); background-size: cover;}
+  75% {background: url(/images/pexels-vishnu-r-nair-1105666.jpg); background-size: cover;}
+  87.5% {background: url(/images/noiseporn-fZGl__1jt3M-unsplash.jpg); background-size: cover;}
+  100% {background: url(/images/tim-toomey-eiY4KJ62P5Q-unsplash.jpg); background-size: cover;}
+}
+
+.gallery {
+  animation: fader 70s cubic-bezier(1,-0.07,.49,.44) infinite;
+}
+
+/* MEDIA QUERY TABLET */
+@media only screen and (min-width: 768px) {
+
+  .background-image {
+    height: 99vh;
+  }
+}
+  
+/* MEDIA QUERY DESKTOP */
+  @media only screen and (min-width: 992px) {
+
+    .background-image {
+      height: 94vh;
+    }
+  }
+
+/* MEDIA QUERY BIGGER DESKTOP */
+
+@media only screen and (min-width: 1200px) {
+
+  .background-image {
+    height: 145vh;
+  }
+}
```

Comment:
Nice that you made the site responsive even though it was not a requirement for this project. As you mentioned about the submit button, I also saw that it clashes with the range slider section but on mobile and tablet the projctes layout and the project overall looks really professional!